src/Illuminate/Encryption/Encrypter.php

```diff
@@ -33,16 +34,23 @@ class Encrypter implements EncrypterContract
      * Create a new encrypter instance.
      *
      * @param  string  $key
-     * @param  string|null  $cipher
+     * @param  string  $cipher
      * @return void
      */
-    public function __construct($key, $cipher = null)
+    public function __construct($key, $cipher = 'AES-128-CBC')
     {
-        $this->key = (string) $key;
+        $len = mb_strlen($key = (string) $key, '8bit');
```

Comment:
https://github.com/laravel/framework/commit/0909b70556471be4fe9d0d96b729a80346d76827#diff-b5d0ee8c97c7abd7e3fa29b9a27d1780

It was there when ext-mbstring was added.